tasks/test.py

```diff
@@ -292,14 +305,7 @@ def test_flavor(
         junit_file_flag = "--junitfile " + junit_file
     args["junit_file_flag"] = junit_file_flag
 
-    for module in modules:
-        module_test_result = ModuleTestResult(path=module.full_path())
-
-        print(f"----- Module '{module.full_path()}'")
-        if not module.condition():
-            print("----- Skipped")
-            continue
-
+    def command(modules_test_results, module, module_test_result):
```

Comment:
```suggestion
    def command(test_results, module, module_result):
```
The old variables name were not really specific. I think we could replace:
- `modules_test_results` with `test_results`
- `module_test_result` with `module_result`

WDYT? If you have a better idea on how to differentiate them, feel free to share it !

Reply:
I applied for the lint_test as well : 
  - `modules_test_results` with `test_results`
  - `module_test_result` with `module_result`
  - `modules_lint_results` with `lint_results`
  - `module_lint_result` with `module_result`
 [d99e2ad](https://github.com/DataDog/datadog-agent/pull/15957/commits/d99e2ad6900a41769652c2c2576935084bc0910a)